docs/layers/identity/deploy.mdx

```diff
@@ -39,12 +39,12 @@ These components are expected to be used together to provide fine-grained role d
     atmos workflow deploy/all -f identity
     ```
 
-    <Note title="IMPORTANT">Once the roles are deployed, the Terraform State Backend should be reconfigured to use the newly provisioned roles.</Note>
+    <Note title="important">Once the roles are deployed, the Terraform State Backend should be reconfigured to use the newly provisioned roles.</Note>
   </Step>
 
   <Step>
     ### <StepNumber/> Reconfigure Terraform State Backend
-    Apply the changes from the previous statep to the Terraform State Backend by running the following:
+    Apply the changes from the previous step to the Terraform State Backend by running the following. This will grant permission for your newly deployed AWS Teams and Team Roles to access the Terraform State Backend.
```

Comment:
```suggestion
    Apply the changes from the previous step to the Terraform State Backend by running the following commands. This will grant permission for your newly deployed AWS Teams and Team Roles to access the Terraform State Backend.
```